libraries/vdf/src/sortition.cpp

```diff
@@ -96,7 +99,7 @@ void VdfSortition::verifyVdf(SortitionParams const& config, bytes const& vrf_inp
       throw InvalidVdfSortition(
           "VDF solution verification failed. Incorrect difficulty. VDF input " + bytes2str(vdf_input) + ", lambda " +
           std::to_string(config.vdf.lambda_bound) + ", difficulty " + std::to_string(getDifficulty()) +
-          ", expected: " + std::to_string(expected) + ", vrf_params: (" + std::to_string(config.vrf.threshold_lower) +
+          ", expected: " + std::to_string(expected) + ", vrf_params: (" + std::to_string(config.vrf.threshold_range) +
```

Comment:
We should change the log because now it is not looks like a range

Reply:
Everywhere in the code we now use upper and range including the config files. I think that it makes sense that it is in the log as well. Maybe we can add some additional text in the log if it is unclear

Reply:
I am just talking about clearer text. Because (num1, num2) looks like a range, but it is not now
I expect something like `vrf_params: range=500, threshold_upper: 1000` would be better